freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/repair/changeprimarygw/ChangePrimaryGatewayService.java

```diff
@@ -62,11 +62,11 @@ public String selectNewPrimaryGatewayInstanceId(Stack stack, List<String> repair
                 }
             }
         } catch (NotFoundException e) {
-            LOGGER.debug("No primary gateway found, searching for an different primary gateway");
+            LOGGER.debug("No primary gateway found, searching for a different primary gateway");
             newPrimaryGatewayInstanceId = assignNewPrimaryGatewayInstanceId(stack, repairInstanceIds);
```

Comment:
@jamisonbennett , Does it make sense to add a follow-up message to know which is the new primary gateway instance?

Reply:
https://github.com/hortonworks/cloudbreak/blob/d67401acd4164577b44bff9a992113ac7556c518/freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/repair/changeprimarygw/ChangePrimaryGatewayService.java#L95
The next log statement which will print for this flow will log "Changing the primary gateway metadata from _ to _" (or it will log "The primary gateway _ metadata did not change" for the case that there isn't a change.

Reply:
It looks like line 71 already has this follow-up message. Let me know if there is something else that you think is needed.
```
        LOGGER.debug("The new primary gateway will be {}", newPrimaryGatewayInstanceId);
```